deploy/stacks/vpc.py

```diff
@@ -108,7 +112,7 @@ def __init__(
                 description=f'{resource_prefix}-{envname}-cidrBlock',
             )
 
-    def create_new_vpc(self, cidr, envname, resource_prefix, restricted_nacl):
+    def create_new_vpc(self, cidr, envname, resource_prefix, restricted_nacl, backend_vpc, tooling_region):
```

Comment:
why do we need those 2 additional parameters?

Reply:
good catch! Will remove